config/sync/user.role.customer.yml

```diff
@@ -7,6 +7,9 @@ label: customer
 weight: 3
 is_admin: null
 permissions:
+  - 'access administration pages'
+  - 'access content overview'
```

Comment:
We should not allow customers access `/admin/content`.
Customers should access workbench pages `/admin/workbench` to manage their content.

Reply:
Cool yep that works.